RockWeb/Themes/NewSpring/assets/Lava/DATASETS/content-channel-items-by-channel.lava

```diff
@@ -176,21 +106,14 @@
         "IsDateVisible": {{ item.ContentChannel | Attribute:'IsDateVisible' | ToJSON }},
         "InteractionChannelId": {{ interactionData | First | Property:'InteractionChannelId' | ToJSON }},
         "InteractionComponentId": {{ interactionData | First | Property:'InteractionComponentId' | ToJSON }},
-        "Title": {{ item.Title | Replace:"'","’" | ToJSON }},
-        "Slug": {{ item.PrimarySlug | ToJSON }},
-        "Status": {{ item.Status | ToJSON }},
-        "Permalink": {{ permalink | Trim | ToJSON }},
-        "PublishDateTime": {{ item.StartDateTime | ToJSON }},
-        "ExpireDateTime": {{ item.ExpireDateTime | ToJSON }},
-        "ModifiedDateTime": {{ item.ModifiedDateTime | ToJSON }},
         "Category": {{ category | ToJSON }},
         "Content": {{ item.Content | Replace:"'","’" | Escape | ToJSON }},
         "Departments": {{ item | Attribute:'Department' | ToJSON }},
-        "Subtitle": {{ item | Attribute:'Subtitle' | ToJSON }},
+        "Subtitle": {{ item | Attribute:'Subtitle' | Replace:"'","’" | ToJSON }},
```

Comment:
Added this filter to string properties - Rock's editor likes to use single quotes instead of apostrophes, which makes it difficult to pass those values through to various lava templates. Adding the filter here makes sure the data is usable at the source.